src/modules/elixir.rs

```diff
@@ -4,9 +4,14 @@ use crate::configs::elixir::ElixirConfig;
 use crate::formatter::StringFormatter;
 use crate::utils;
 
-use once_cell::sync::Lazy;
+use async_once::AsyncOnce;
+use lazy_static::lazy_static;
 use regex::Regex;
-use std::ops::Deref;
+
+lazy_static! {
+    static ref VERSIONS: AsyncOnce<Option<(String, String)>> = AsyncOnce::new(get_elixir_version());
+}
```

Comment:
I've just seen that utils/Context PR merge, indeed, it's going to need some reworking.

So you're saying it's ok to depend on `futures`, in addition to `async_std`, correct? I had actually tried to avoid adding it in the first pass :smile:

Reply:
I adding it should be fine, but I don't have much experience with async in rust. I didn't really notice much of an impact on compile times, is there any other reason to avoid it?

Reply:
I'm also not seeing increased compile times. The idea was just to keep dependencies focused, but otherwise there shouldn't be any issues. It will actually improve some areas.